parsl/monitoring/db_manager.py

```diff
@@ -544,7 +544,11 @@ def start(self,
                 if reprocessable_last_resource_messages:
                     self._insert(table=STATUS, messages=reprocessable_last_resource_messages)
             except Exception:
-                logger.exception("Exception in db loop: this might have been a malformed message, or some other error. monitoring data may have been lost")
+                log_message = (
```

Comment:
Hi, thanks for your PR. You could avoid using an intermediate variable here. You could use multi-line string syntax directly on the method call to logger.exception. Here's an example function call like that to show what I mean:

```
def f(x: str):
 print(x)

f('a'
  'b')
```